plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -15,14 +15,15 @@ import (
 
 // SQLServer struct
 type SQLServer struct {
-	Servers       []string `toml:"servers"`
-	QueryVersion  int      `toml:"query_version"`
-	AzureDB       bool     `toml:"azuredb"`
-	DatabaseType  string   `toml:"database_type"`
-	IncludeQuery  []string `toml:"include_query"`
-	ExcludeQuery  []string `toml:"exclude_query"`
-	queries       MapQuery
-	isInitialized bool
+	Servers             []string `toml:"servers"`
+	QueryVersion        int      `toml:"query_version"`
+	AzureDB             bool     `toml:"azuredb"`
+	DatabaseType        string   `toml:"database_type"`
+	IncludeQuery        []string `toml:"include_query"`
+	ExcludeQuery        []string `toml:"exclude_query"`
+	IgnoreDefaultServer bool     `toml:"ignore_default_server"`
```

Comment:
This should be a separate PR or else will get lost in the HADR changes, this has nothing to do with HADR right?

Reply:
Makes sense, pulled out last commit to only have HADR changes.